doc/available_metrics.md

```diff
@@ -5,9 +5,11 @@ All measurements share the following metrics:
 
 | Metric | Type | Description | Introduced in |
 | --- | --- | --- | --- |
+| instance | string | Jenkins instance url | 3.4 |
 | build_number | integer | Build number |  |
 | project_name | string | Build name |  |
 | project_path | string | Build path | 1.15 |
+| project_namespace | string | Root path to the project (parent folder) | 3.4 |
```

Comment:
This seems to acutally be the root folder instead of parent folder. E.g with my dev env I have a structure `influxdb-plugin/pipelines/some-job` the value of `project_namespace` will be `influxdb-plugin` (root folder) instead of `pipelines` (parent folder).